twofish/LICENSE-APACHE

```diff
@@ -0,0 +1,13 @@
+Copyright 2017 Damian Czaja
```

Comment:
I think it's better to take LISENCE-APACHE which is use across other crates. (e.g. from streebog) I've missed that rc2 crate uses a different file.